src/time.jsx

```diff
@@ -45,7 +49,7 @@ export default class Time extends React.Component {
 
   state = {
     height: null
-  }
+  };
```

Comment:
```suggestion
  };
```

Please review project style